testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/transaction/SLSB1.java

```diff
@@ -61,7 +61,7 @@ public Employee createEmployee(String name, String address, int id) {
 
     /**
      * Makes two DAO calls, the transaction fails during the first DAO call. The
-     * JTA transaction is rolled back and no database changes should occur.
+     * Jakarta Transactions rolled back and no database changes should occur.
```

Comment:
Jakarta Transaction is

Reply:
made it `Jakarta Transactions transaction is rolled back and no database changes should occur.`